src/components/AAPModal/AnsibleAutomationPlatformModal.tsx

```diff
@@ -188,7 +256,10 @@ const AAPModal = ({ onClose, initialStatus }: Props) => {
       const data = await getAAPData(signupData.defaultUserNamespace);
       const status = getReadyCondition(data, handleSetAAPCRError);
       setStatus(status);
-      if (data != undefined && data.items.length > 0 && data.items[0].status != undefined) {
+      if (data == undefined || data.items.length <= 0) {
```

Comment:
@mfrancisc is it possible that `data.items` would ever be undefined?

Reply:
in theory no, it's a list/array so it should be empty, but I'm not expert at typescript :) . Anyways I've added an additional check in https://github.com/RedHatInsights/developer-sandbox-ui/pull/67/commits/836ef45b9442fd2c459a735a5b09f710e7b1c76e